tests/tape/test_tape_measure.py

```diff
@@ -471,13 +471,17 @@ def func():
     @pytest.mark.parametrize(
         "device", ["default.qubit", "default.qubit.tf", "default.qubit.autograd"]
     )
-    def test_devices(self, device, skip_if_no_tf_support):
+
+    @pytest.mark.parametrize(
+        "diff_method", ["best", "finite-diff", "parameter-shift"]
+    )
+    def test_devices(self, device, diff_method, skip_if_no_tf_support):
         """Test that the returned state is equal to the expected returned state for all of
         PennyLane's built in statevector devices"""
 
         dev = qml.device(device, wires=4)
 
-        @qnode(dev, diff_method="parameter-shift")
+        @qnode(dev, diff_method=diff_method)
```

Comment:
`"best"` may cover `"backprop"` too for relevant devices and tests the new behaviour.